sdk/communication/communication-job-router/README.md

```diff
@@ -0,0 +1,396 @@
+| Exception Policy | A container of rules that define what action to take when a particular exception occurs.  This container of rules can be associated with one or more queues. |
+
+##Setting Up
+###Install an IDE such as [VSCode](https://code.visualstudio.com/download) or [Webstorm](https://www.jetbrains.com/webstorm/download/) if you haven't (Optional)
+
+###Install NodeJS if you haven't
+LTS or Current version, either is ok.
+https://nodejs.org/en/download/
+
+###Create a new NodeJS Express server
+In a console window (such as cmd, PowerShell, or Bash), create a new folder named RouterQuickStart and use `npx express-generator` to create a new Express app in the current folder. This will create a simple "Hello World" Express project that will listen on port 3000
+```
+npx express-generator
+npm install
+DEBUG=routerquickstart:* npm start
+```
+
+###Create an ACS resource under dogfood
+Refer to this doc, https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/20243/Azure-dogfood-env
+Create a new resource or use existing resource. If creating a resource, select acs-pstn-dogfood as resource group.
+
+![image.png](/.attachments/image-1a0118bb-0063-420b-924a-943b66ef0d01.png)
+
+###Install the Azure Communication Services Router SDK
+Create a file .npmrc to your project. Put following code into it:
+```
+@azure-ccaap:registry=https://pkgs.dev.azure.com/azure-sdk/internal/_packaging/azure-sdk-for-js-pr/npm/registry/ 
+registry=https://registry.npmjs.org
+                        
+always-auth=true
+```
+
+Then, run vsts-npm-auth to get an Azure Artifacts token added to your user-level .npmrc file
+
+`vsts-npm-auth -config .npmrc`
+
+If it says vsts-npm-auth not found, then you need to install it first:
+
+`npm install -g vsts-npm-auth`
+
+
+While still in the application directory, install the Azure Communication Services Router client library for Javascript package by using the npm install --save command
+`npm install --save @azure-ccaap/communication-router`
+
+## Examples
+###Initialize Router Client
+First we need to initialize a Router client.
+```js
+import { RouterClient } from '@azure/communication-router';
+
+const acsConnectionString = 'endpoint=https://<YOUR_ACS>.communication.azure.com/;accesskey=<YOUR_ACCESS_KEY>'
+const routerClient = new RouterClient(acsConnectionString );
+```
+
+##Configure Queue and Workers
+###Create a queue
+Then we create a sales queue with various labels.
+```js
+const salesQueueResponse = await routerClient.CreateQueue({
+  "name": "Sales",
+  "distributionPolicyId": "0832f57a-a651-4cd3-b721-1a1ce76ecf2b",
+  "labels": {
+    "Department": "Xbox"
+  },
+  "exceptionPolicyId": "cb272ee4-38ae-41d8-9408-190d9c7a98aa"
+});
+```
+
+###Register workers
+Register workers “Bob” and “Alice” with various abilities.
+```js
+// Register worker Bob
+const bobWorkerResponse = await routerClient.RegisterWorker({
+  "id": "21837c88-6967-4078-86b9-1207821a8392",
```

Comment:
It feels weird to see quoted property names of a plain object in a JavaScript script. What about removing the quotes from the property names?

Reply:
Keep in mind that agreeing to do this change once means agreeing to do this change everywhere in this file 🙂

Reply:
I would keep it, we have user-defined string as property keys, for example:
```
  "abilities": {
    "Xbox": 5,
    "English": 3
  },
```